ui/Screen/NetworkDiagnostics.svelte

```diff
@@ -0,0 +1,67 @@
+<script lang="typescript">
+  import Router from "svelte-spa-router";
+
+  import { store } from "../src/screen/network";
+  import * as path from "../src/path";
+
+  import ActionBar from "../DesignSystem/Component/ActionBar.svelte";
```

Comment:
We can use absolute imports now.
```suggestion
  import ActionBar from "ui/DesignSystem/Component/ActionBar.svelte";
```
It would be great if you could make a quick pass and change it everywhere a path goes up a directory with `../`.